src/seidr.py

```diff
@@ -0,0 +1,70 @@
+import numpy as np
+from scipy.integrate import odeint
+import matplotlib.pyplot as plt
+
+
+# The SIR model differential equations.
+def SEIDR_deriv(y, t, N, beta, gamma, mi, lambda_, epsilon, alpha):
+    S, E, I, D, R = y
+    dSdt = (-beta * S * I / N) -  mi*S +lambda_
+    dEdt = beta * S * I / N - (mi+epsilon)*E
+    dIdt =  epsilon*E - (gamma+mi+alpha)*I
+    dDdt = alpha*I
+    dRdt = gamma * I - mi*R
+    return dSdt, dEdt, dIdt, dDdt, dRdt
+
+def SEIDR(initial_conditions, no_of_days, N, beta, gamma, mi, lambda_, epsilon, alpha):
```

Comment:
what about implementing it as a class? So we would have a class named `SEIDR` and inside this class, we would have methods like `simulate` or `plot`, without the need of passing all those parameters all the time.